test/test_response.c

```diff
@@ -0,0 +1,206 @@
+#include "unity.h"
+#include <msgpack.h>
+//#define LOG_LOCAL_LEVEL ESP_LOG_DEBUG
+#include <esp_log.h>
+#include "response.h"
+#include "message.h"
+#include "storage.h"
+#include "ubirch_protocol.h"
+#include <ubirch_ed25519.h>
+
+static const char *TAG = "UBIRCH API TEST RESPONSE";
+
+//
+#define PREVIOUS_SIGNATURE_START (4 + 16 + 2)
+
+// test signing key
+unsigned char test_ed25519_public_key[] = {
+        0x2f, 0x17, 0xc2, 0x3a, 0x62, 0x6d, 0x4e, 0x78,
+        0x04, 0xde, 0x1d, 0xb1, 0x96, 0x91, 0x93, 0x9c,
+        0xe7, 0xe9, 0x7e, 0xe1, 0x5c, 0xcd, 0x94, 0x25,
+        0x4a, 0xb3, 0xde, 0xea, 0xd9, 0x7c, 0xfe, 0x22
+};
+
+// "backend signing key"
+unsigned char server_public_key[] = {
+    0xef, 0x80, 0x48, 0xad, 0x06, 0xc0, 0x28, 0x5a,
+    0xf0, 0x17, 0x70, 0x09, 0x38, 0x18, 0x30, 0xc4,
+    0x6c, 0xec, 0x02, 0x5d, 0x01, 0xd8, 0x60, 0x85,
+    0xe7, 0x5a, 0x4f, 0x00, 0x41, 0xc2, 0xe6, 0x90
+};
+
+
+void response_handler(const void* data, const size_t len) {
+    ESP_LOGD(TAG, " response handler called");
+    ESP_LOG_BUFFER_HEXDUMP("response handler data", data, len, ESP_LOG_DEBUG);
+}
+
+static int ed25519_verify_test0(const unsigned char *data, size_t len,
+        const unsigned char signature[UBIRCH_PROTOCOL_SIGN_SIZE]) {
+    return ed25519_verify_key(data, len, signature, test_ed25519_public_key);
+}
+
+static int ed25519_verify_test1(const unsigned char *data, size_t len,
```

Comment:
maybe rename to `ed25519_verify_signature_from_backend_test(...)`?

Reply:
Done.